packages/emmett-fastify/src/index.ts

```diff
@@ -0,0 +1,101 @@
+import Compress from '@fastify/compress';
+import Etag from '@fastify/etag';
+import Form from '@fastify/formbody';
+import closeWithGrace from 'close-with-grace';
+import Fastify, {
+  type FastifyInstance,
+  type FastifyReply,
+  type FastifyRequest,
+} from 'fastify';
+
+const defaultPlugins = [
+  { plugin: Etag, options: {} },
+  { plugin: Compress, options: { global: false } },
+  { plugin: Form, options: {} },
+];
+
+const defaultPostMiddlewares = (app: FastifyInstance) => {
+  app.all('*', async (request: FastifyRequest, reply: FastifyReply) => {
```

Comment:
**CONCERN:** Shouldn't we use `setNotFoundHandler` here? https://fastify.dev/docs/latest/Reference/Server/#setnotfoundhandler

Also shouldn't we have here also EmmetError mapping? I think that Fastify maps automatically `statusCode` field from error, and EmmettError has `errorCode` instead, so I think that it won't be automatically mapped.

Reply:
I will remove this to allow user use handling error how they want.